exec/exec-manager-provider.go

```diff
@@ -77,7 +77,7 @@ func CreateExecManager() ExecManager {
 
 		return docker_infra.New(dockerClient, containerFilter, shellDetector)
 	default:
-		log.Fatal("Unable to create manager for current infrastructure.")
+		log.Println("Error: Unable to create manager for current infrastructure.")
```

Comment:
Is this message visible for a user on Theia IDE?

Reply:
No.

Reply:
Can you then describe or attach a screencast: what user will see if unsupported infra is used?